tests/e2e/matmul/BUILD.bazel

```diff
@@ -569,7 +569,61 @@ iree_generated_e2e_runner_test(
         "requires-gpu-cdna3",
     ],
     target_backends_and_drivers = [
-        ("rocm", "rocm"),
+        ("rocm", "hip"),
+    ],
+    test_runner = "//tools/testing/e2e:iree-e2e-matmul-test",
+    test_type = "matmul",
+)
+
+# Testing RDNA3 + matrix core path.
+iree_generated_e2e_runner_test(
+    name = "e2e_matmul_rocm_f16_large_rdna3_wmma",
+    compiler_flags = [
+        "--iree-rocm-target-chip=gfx1100",
+    ],
```

Comment:
On https://github.com/iree-org/iree/tree/e2e-tests-matmul that @erman-gurses is working on, any targets using `--iree-rocm-target-chip` were removed from Bazel and added to CMake with the `IREE_HIP_TEST_TARGET_CHIP` option. There may be some merge conflicts to resolve between this PR and that work.

We could try to support both Bazel and CMake, but I'd rather focus on just CMake unless there is a specific need for Bazel + ROCm. The requirement to set the target chip is tricky to support across both.

Reply:
Thanks for that update. Yes, I can make changes accordingly on my side.